### ISSUE TITLE
Misleading pickers doc - DatePickerDialog and TimePickerDialog doesn't implement material design on pre-Lollipop devices

### ISSUE THREAD

**Issue by gregkorossy** [materialdoc/materialdoc-web#10]
The docs are a bit misleading about the look-and-feel of the pickers because the appcompat lib implies that they will have the material treatment on all compatible platforms, but this is not the case.

This is what it looks like on API 15 using `android.app.*`:

DatePickerDialog | TimePickerDialog
-|-
![screenshot_1504549444](https://user-images.githubusercontent.com/5181621/30035887-0389b89a-91af-11e7-966c-14058a410f38.png) | ![screenshot_1504550043](https://user-images.githubusercontent.com/5181621/30036092-67b81072-91b0-11e7-98b6-f6e1f0e367d4.png)

---

And this is how it should (_this is the (8.0 - Oreo) framework (**not** the "opt") version [backported](https://github.com/Gericop/DateTimePicker) to API 14+; I made it because I didn't like the old look and wanted to provide material on all compatible API levels_):

DatePickerDialog | TimePickerDialog
-|-
![screenshot_1504549535](https://user-images.githubusercontent.com/5181621/30035917-3b0f2732-91af-11e7-847d-b7ab95f2ec0c.png) | ![screenshot_1504549946](https://user-images.githubusercontent.com/5181621/30036078-48baed34-91b0-11e7-951e-73864acfab5e.png)

Maybe a little warning would be "nice", so people wouldn't expect them to have the material design on pre-L devices?

**Comment by dmytrodanylyk** [comment]
Hi,

Thanks for your reply. We are still deciding about what min API we should target to provide documentation.

In my opinion it's time to move to `v21`, most of new projects will start with `minSDK 21`. It would be easier to keep documentation up to date.

**Comment by gregkorossy** [comment]
I'm not sure the Android ecosystem is ready for minSdk 21 yet (I wish it was). Even Google devs don't think so as they've backported the fragment transitions (shared elements) to API 14+ (_should've been part of support lib 26.0.0, but they haven't released it yet for whatever reasons; see [this](https://plus.google.com/110757386741030050881/posts/gzzp2MZ1Wgn) discussion, especially the latest comments_). Also, if you take a look at the [dashboard](https://developer.android.com/about/dashboards/index.html#Platform), you'll see that API 15-19 represent a whopping 24.3% of the devices that visited the Play Store during that given period, which is, in my opinion, still a significant amount.

Of course it would be nice if we could just use minSdk 21, but I think we're not there yet. Nonetheless, these **awesome** collections of docs and guides can support API 21+ only, but it should clearly state so people would know before implementing some not-everywhere-material stuff only to find out from dissatisfied users.

**Comment by dmytrodanylyk** [comment]
@Gericop I agree. We will figure out if we will support <21 or >=21 and put some note on home page, thanks.

Before that happens ticket will be open.
